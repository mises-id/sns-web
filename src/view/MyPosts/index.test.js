/*
 * @Author: lmk
 * @Date: 2021-07-16 00:14:28
 * @LastEditTime: 2021-07-16 00:14:43
 * @LastEditors: lmk
 * @Description: MyPost test
 */

import React from 'react';
import MyPost from './';
import {shallow } from 'enzyme';
describe('MyPost', () => {
  it('should render correctly', () => {
    const component = shallow(<MyPost />)
    expect(component).toMatchSnapshot();
  });
});