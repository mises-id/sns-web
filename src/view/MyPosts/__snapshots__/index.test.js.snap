// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyPost should render correctly 1`] = `
<div>
  <Navbar
    title="myPostPageTitle"
  />
  <ForwardRef(PullList)
    data={Array []}
    load={[Function]}
    renderView={[Function]}
  />
  <div
    className="m-position-fixed createPosts"
  >
    <Image
      onClick={[Function]}
      size={75}
      source="send.png"
    />
  </div>
  <ForwardRef(forwardRef)
    actions={
      Array [
        Object {
          "onClick": [Function],
          "text": "Delete",
          "theme": "danger",
        },
      ]
    }
    onCancel={[Function]}
    onMaskClick={[Function]}
    spacing={true}
    visible={false}
  />
</div>
`;
